src/core/wallets/bbn/injectable/index.ts

```diff
@@ -0,0 +1,15 @@
+import { Network, type BBNConfig, type WalletMetadata } from "@/core/types";
+
+import { BBNProvider } from "../BBNProvider";
+
+const metadata: WalletMetadata<BBNProvider, BBNConfig> = {
+  id: "injectable",
+  name: (wallet) => wallet.getWalletProviderName?.(),
+  icon: (wallet) => wallet.getWalletProviderIcon?.(),
+  docs: "",
+  wallet: "bbnwallet",
+  createProvider: (wallet) => wallet,
+  networks: [Network.MAINNET, Network.SIGNET],
```

Comment:
@gbarkhatov don't forget to add new metadata parameter `label`

Reply:
Check comment here https://github.com/babylonlabs-io/bbn-wallet-connect/pull/113#discussion_r1882184665